test/BDNPerfTests/ConfigFiles/CI_CONFIG_BDN_Benchmark_Cluster.ClusterOperations.json

```diff
@@ -0,0 +1,31 @@
+{
+    "configuration": "Release",
+    "framework": "net8.0",
+    "filter": "BDN.benchmark.Cluster.ClusterOperations.*",
+
+    "expectedGet_DSV_AllocatedValue_linux": 0,
+    "expectedSet_DSV_AllocatedValue_linux": 0,
+    "expectedMGet_DSV_AllocatedValue_linux": 0,
+    "expectedMSet_DSV_AllocatedValue_linux": 0,
+    "expectedCPBSet_DSV_AllocatedValue_linux": 0,
+    "expectedGet_None_AllocatedValue_linux": 0,
+    "expectedSet_None_AllocatedValue_linux": 0,
+    "expectedMGet_None_AllocatedValue_linux": 0,
+    "expectedMSet_None_AllocatedValue_linux": 0,
+    "expectedCPBSet_None_AllocatedValue_linux": 0,
+
+    "expectedGet_DSV_AllocatedValue_win": 0,
+    "expectedSet_DSV_AllocatedValue_win": 0,
+    "expectedMGet_DSV_AllocatedValue_win": 0,
+    "expectedMSet_DSV_AllocatedValue_win": 0,
+    "expectedCPBSet_DSV_AllocatedValue_win": 0,
+    "expectedGet_None_AllocatedValue_win": 0,
+    "expectedSet_None_AllocatedValue_win": 0,
+    "expectedMGet_None_AllocatedValue_win": 0,
+    "expectedMSet_None_AllocatedValue_win": 0,
+    "expectedCPBSet_None_AllocatedValue_win": 0,
+
+    "acceptableAllocatedRange": 10, 
```

Comment:
make these three lines part of the script, rather than put it in every file. it should be very stable. this change will be needed in order to get rid of the entire file anyway, per earlier comment.

Reply:
This gives us some flexibility to have the acceptable range linked to the specific BDN. If we move it to the script, then all BDN runs use the same value. I guess it is ok and we already do that for the Mean values, but it just loses some flexibility.

Reply:
Done